intern/remote/boss/tests/int_test_volume_v1.py

```diff
@@ -912,5 +912,32 @@ def test_upload_and_download_to_channel_4D(self):
         actual = self.rmt.get_cutout(self.chan, 0, x_rng, y_rng, z_rng, time_range=t_rng)
         numpy.testing.assert_array_equal(data, actual)
 
```

Comment:
Can you add a test that "blacks" out a smaller portion of the original upload?

Reply:
add this on commit 52b3949, let me know if I should add any more!